src/guide/payment_notification.md

```diff
@@ -0,0 +1,115 @@
+
+## Step 1: Run your own NDS
+As written above you will need to run your own NDS because NDS is configured to send push notifications to your app users, therefore configured with the required keys and certificates.
+If you don't want to write it from scratch you can use our running <a href="https://github.com/breez/notify">NDS</a>
+
+Our used NDS expects urls in the following format:
+<section><i>https://your-nds-service.com/notify?platform=<ios|android>&token=[PUSH_TOKEN]</i></section>
+
+Once the NDS received such request it will send a push notification to the matched device.
+
+## Step 2: Obtain a mobile push token
+First, ensure that your mobile application is set up to receive push notifications and can generate a push token. This token uniquely identifies the device for push notifications.
+
+* For iOS, use Apple Push Notification Service (APNs) to get the token.
+* For Android, use Firebase Cloud Messaging (FCM) to obtain the token.
+
+## Step 3: Register the Webhook with Breez
+Register the constructed URL with the Breez SDK By calling an SDK function.
+
+<custom-tabs category="lang">
+<div slot="title">Rust</div>
+<section>
+
+```rust,ignore
+{{#include ../../snippets/rust/src/webhook.rs:register-payment-webook}}
+```
+</section>
+</custom-tabs>
+
+## Step 4: Handling Notifications When the App is Not Running
+To ensure that your mobile application can handle payment notifications even when it is not actively running, specific implementations are required for both iOS and Android platforms. This involves waking up the app upon receiving a push notification, connecting with the Breez SDK, and then waiting for the payment to be fully received.
+
+### iOS Implementation Using NotificationServiceExtension
+
+#### Overview
+For iOS, the app must use NotificationServiceExtension to process notifications in the background.
+
+#### Steps for Implementation
+* Integrate NotificationServiceExtension: Add NotificationServiceExtension to your iOS project. This extension allows your app to process the incoming push notification data in the background before presenting it to the user.
+
+* Wake Up the App: When a push notification is received, the NotificationServiceExtension will be triggered, waking up the app.
+
+* Connect with Breez SDK: Inside the extension, implement the logic to establish a connection with the Breez SDK which also runs the signer so the payment can be processed.
```

Comment:
Note: This is the first mention of the signer in the documentation. We should introduce this entire concept of having to be online in order to receive a payment in the documentation elsewhere.

Reply:
I used different wording for now